models.rdf

```diff
@@ -16,8 +16,8 @@
       <dcterms:publisher rdf:resource="http://www.duraspace.org/"/>
       <rdfs:seeAlso rdf:resource="https://github.com/duraspace/pcdm/wiki"/>
       <rdfs:comment xml:lang="en">Ontology for the Portland Common Data Model, intended to underlie a wide array of repository and DAMS applications.</rdfs:comment>
-      <owl:versionInfo>2016/04/18</owl:versionInfo>
-      <owl:priorVersion rdf:resource="http://pcdm.org/2015/09/28/models"/>
+      <owl:versionInfo>2016/09/01</owl:versionInfo>
```

Comment:
And this: https://github.com/duraspace/pcdm/blob/27a133e2f5cdbf23cf0b2c125b2eb7f086c418bb/models.rdf#L15